Detectors/ITSMFT/ITS/tracking/src/IOUtils.cxx

```diff
@@ -166,6 +217,53 @@ int ioutils::loadROFrameData(const o2::itsmft::ROFRecord& rof, ROframe& event, g
   return clusters_in_frame.size();
 }
 
+int ioutils::loadROFrameData(const o2::itsmft::ROFRecord& rof, ROframe& event, gsl::span<const itsmft::CompClusterExt> clusters, std::vector<unsigned char>::const_iterator& pattIt, const itsmft::TopologyDictionary& dict,
+                             const dataformats::MCTruthContainer<MCCompLabel>* mcLabels)
+{
+  event.clear();
+  GeometryTGeo* geom = GeometryTGeo::Instance();
+  geom->fillMatrixCache(utils::bit2Mask(TransformType::T2L, TransformType::L2G));
+  int clusterId{0};
+
+  auto first = rof.getFirstEntry();
+  auto clusters_in_frame = rof.getROFData(clusters);
+  for (auto& c : clusters_in_frame) {
+    int layer = geom->getLayer(c.getSensorID());
+
+    auto pattID = c.getPatternID();
+    Point3D<float> locXYZ;
+    float sigmaY2 = 0.0015 * 0.0015, sigmaZ2 = sigmaY2, sigmaYZ = 0; //Dummy COG errors (about half pixel size)
+    if (pattID == itsmft::CompCluster::InvalidPatternID || dict.IsGroup(pattID)) {
+      o2::itsmft::ClusterPattern patt(pattIt);
+      locXYZ = dict.getClusterCoordinates(c, patt);
```

Comment:
and here

Reply:
@OK.  I will get the errors from the groups.